pages/instructorAssessmentManualGrading/assessment/assessment.sql

```diff
@@ -1,5 +1,16 @@
 -- BLOCK select_questions_manual_grading
-WITH instance_questions_with_submission AS (
+WITH
+selected_instance_questions AS (
+    SELECT DISTINCT ON (iq.id)
+        iq.*
```

Comment:
With the whole "CTEs are optimization boundaries" thing, would we benefit from being more selective with which columns we actually read here? I'll admit I don't have a great understanding of how Postgres is actually executing this.

Reply:
My understanding is that it is materializing the CTE (storing it in a temporary table) so it probably _would_ be more efficient, as you say.

I think @jonatanschroeder is working on an alternative fix, so let me wait on that before I mess around with this one any further.

Reply:
In essence this entire subquery can be replaced with checking if status is different than unanswered. The status is set to unanswered on insert (default), and then changed to saved when a submission is created, and never changed back to unanswered. The only difference would be if there were a way to delete submissions, which I don't think there is at this point.